{
  "id": "CVE-2021-29719",
  "assigner": "psirt@us.ibm.com",
  "Published": "2021-12-03T16:15:10.103Z",
  "Modified": "2022-01-04T15:12:17.303Z",
  "last-modified": "2022-01-04T15:12:17.303Z",
  "summary": "IBM Cognos Analytics 11.1.7 and 11.2.0 could be vulnerable to client side vulnerabilties due to a web response specifying an incorrect content type. IBM X-Force ID: 201091",
  "access": {
    "authentication": "NONE",
    "complexity": "LOW",
    "vector": "NETWORK"
  },
  "impact": {
    "availability": "NONE",
    "confidentiality": "PARTIAL",
    "integrity": "NONE"
  },
  "cvss": 5.0,
  "exploitabilityScore": 10.0,
  "impactScore": 2.9,
  "cvss-vector": "AV:N/AC:L/Au:N/C:P/I:N/A:N",
  "references": [
    "https://exchange.xforce.ibmcloud.com/vulnerabilities/201091",
    "https://security.netapp.com/advisory/ntap-20211223-0006/",
    "https://www.ibm.com/support/pages/node/6520510"
  ],
  "vulnerable_configuration": [
    "cpe:2.3:a:ibm:cognos_analytics:11.1.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.1:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.2:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.3:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.4:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.5:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.6:*:*:*:*:*:*:*"
  ],
  "vulnerable_product": [
    "cpe:2.3:a:ibm:cognos_analytics:11.1.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.1:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.2:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.3:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.4:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.5:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.6:*:*:*:*:*:*:*"
  ],
  "vendors": [
    "ibm"
  ],
  "products": [
    "cognos_analytics"
  ],
  "vulnerable_product_stems": [
    "cpe:2.3:a:ibm:cognos_analytics"
  ],
  "vulnerable_configuration_stems": [
    "cpe:2.3:a:ibm:cognos_analytics"
  ],
  "cwe": "CWE-668",
  "vulnerable_configuration_cpe_2_2": []
}